app/src/main/java/edu/ucsc/cmps115_spring2017/face2name/MainScreen.java

```diff
@@ -2,18 +2,23 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.view.MotionEvent;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.util.Log;
 
 import edu.ucsc.cmps115_spring2017.face2name.Camera.CameraPreview;
 import edu.ucsc.cmps115_spring2017.face2name.Camera.OrientationCapability;
 import edu.ucsc.cmps115_spring2017.face2name.Camera.OrientationCapability.OrientationSetting;
 
+import edu.ucsc.cmps115_spring2017.face2name.Layer.LayerView;
 
-public class MainScreen extends AppCompatActivity implements CameraPreview.PreviewCallbacks {
+
+public class MainScreen extends AppCompatActivity implements CameraPreview.PreviewCallbacks{
```

Comment:
This whitespace change doesn't add anything, and it makes for inconsistent formatting.